python/cudf/cudf/tests/test_string.py

```diff
@@ -611,35 +572,16 @@ def test_string_index_str_cat(data, others, sep, na_rep, name):
 
 
 @pytest.mark.parametrize(
-    "data",
-    [
-        pytest.param(
-            ["a", None, "c", None, "e"],
-            marks=pytest.mark.xfail(
-                reason="https://github.com/rapidsai/cudf/issues/5862"
-            ),
-        ),
-        ["a", "b", "c", "d", "a"],
-    ],
```

Comment:
version

Reply:
Did not version this as it was a cudf bug that got resolved.